src/main/scala/shine/C/AST/Nodes.scala

```diff
@@ -61,6 +61,8 @@ abstract class TypedefDecl(val t: Type, override val name: String) extends Decl(
 abstract class StructTypeDecl(override val name: String,
                               val fields: Seq[VarDecl]) extends Decl(name)
 
+abstract class IncludeDecl(override val name: String) extends Decl(name)
```

Comment:
I am also adding Include directives in another PR, we can synchronize our designs https://github.com/rise-lang/shine/blob/d17c1fa34758c069ee37f137a73410c8fe87370d/src/main/scala/shine/C/AST/package.scala#L9-L16

Reply:
I only added this IncludeDecl, but not start to use it. So i suggest to remove my IncludeDecl and use sometime your include directives.